templates/zerver/help/change-a-user's-name.md

```diff
@@ -0,0 +1,25 @@
+# Change a user's name
+Name of the user can be changed by the admin as well as user himself.
+
+Following are the steps to change the name of the user using an admin account:
+* Click on the ![cog](/static/images/help/cog.png) icon in the upper right corner
+of the right sidebar
+
+* Choose `Administration` in the ![admin-panel](/static/images/help/admin.png) panel.
+
+* The following window will be invoked:
+![admin-window](/static/images/help/admin-window.png)
+
+* Click on the tab named `Users` and you will find a window like the below one:
+![user-name-window](/static/images/help/user-name-window.png)
+
+* Click on the ![pencil](/static/images/help/pencil.png) icon next to the name of the user
+you want to change.
+
+* You will find the name of the user editable as shown below:
```

Comment:
That sounds really weird, how about "You can edit the user's name as shown below"?